nucypher/network/retrieval.py

```diff
@@ -182,13 +183,14 @@ class PRERetrievalClient(ThresholdAccessControlClient):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def _request_reencryption(self,
-                              ursula: 'Ursula',
-                              reencryption_request: ReencryptionRequest,
-                              alice_verifying_key: PublicKey,
-                              policy_encrypting_key: PublicKey,
-                              bob_encrypting_key: PublicKey,
-                              ) -> Dict['Capsule', 'VerifiedCapsuleFrag']:
+    def _request_reencryption(
+        self,
+        ursula: "lawful.Ursula",
+        reencryption_request: ReencryptionRequest,
+        alice_verifying_key: PublicKey,
+        policy_encrypting_key: PublicKey,
+        bob_encrypting_key: PublicKey,
+    ) -> Dict["Capsule", "VerifiedCapsuleFrag"]:
```

Comment:
These are already imported:

```suggestion
    ) -> Dict[Capsule, VerifiedCapsuleFrag]:
```

Reply:
This is from 3159 which has already been merged